docs/streams/developer-guide/dsl-api.html

```diff
@@ -1651,13 +1658,11 @@ <h4><a id="streams_concepts_globalktable" href="#streams_concepts_globalktable">
                     <p>Each case is explained in more detail in the subsequent sections.</p>
                     <div class="section" id="join-co-partitioning-requirements">
                         <span id="streams-developer-guide-dsl-joins-co-partitioning"></span><h5><a class="toc-backref" href="#id14">Join co-partitioning requirements</a><a class="headerlink" href="#join-co-partitioning-requirements" title="Permalink to this headline"></a></h5>
-                        <p>Input data must be co-partitioned when joining. This ensures that input records with the same key, from both sides of the
+                        <p>For most joins, Input data must be co-partitioned when joining. This ensures that input records with the same key, from both sides of the
                             join, are delivered to the same stream task during processing.
-                            <strong>It is the responsibility of the user to ensure data co-partitioning when joining</strong>.</p>
-                        <div class="admonition tip">
-                            <p><b>Tip</b></p>
-                            <p class="last">If possible, consider using <a class="reference internal" href="#streams_concepts_globalktable"><span class="std std-ref">global tables</span></a> (<code class="docutils literal"><span class="pre">GlobalKTable</span></code>) for joining because they do not require data co-partitioning.</p>
-                        </div>
+                            <strong>It is the responsibility of the user to ensure data co-partitioning when joining</strong>.
```

Comment:
Nit: I always think it's strange to refer to "the user" in the docs... The explicit audience of documentation is the user, so it's both more appropriate and more natural to use second person pronouns.

```suggestion
                            <strong>It is your responsibility to ensure data co-partitioning when joining</strong>.
```